### ISSUE TITLE
JS error in catalog-add-to-cart-mixin.js when adding products to cart

### ISSUE THREAD

**Issue by simonrl** [FACT-Finder-Web-Components/magento2-module#346]
- FACT-Finder-Web-Components Magento2-Module version (see composer.json): _tested on 2.1 and 2.2_
- Magento edition and version: _Commerce 2.3.5-p2_
- PHP version: _7.3_
- Expected behavior: _Ajax add to cart button should not throw JS error_
- Actual behavior: _Ajax add to cart button throws JS error_
- Steps to reproduce: _Click on ajax add to cart button_

The JS error:
```
catalog-add-to-cart-mixin.min.js:1 Uncaught TypeError: Cannot read properties of undefined (reading 'id')
    at HTMLDocument.<anonymous> (catalog-add-to-cart-mixin.min.js:1)
    at HTMLDocument.dispatch (jquery.min.js:338)
    at HTMLDocument.elemData.handle (jquery.min.js:311)
    at Object.trigger (jquery.min.js:332)
    at Object.jQuery.event.trigger (jquery-migrate.min.js:54)
    at HTMLDocument.<anonymous> (jquery.min.js:380)
    at Function.each (jquery.min.js:28)
    at jQuery.fn.init.each (jquery.min.js:16)
    at jQuery.fn.init.trigger (jquery.min.js:380)
    at Object.success (catalog-add-to-cart.min.js:2)
```

**Comment by a-laurowski** [comment]
Hi @simonrl 
I think i see what the problem is. The Magento2.3.5 catalog-add-to-cart.js sends this

         $(document).trigger('ajax:addToCart', {
                        'sku': form.data().productSku,
                        'productIds': productIds,
                        'form': form,
                        'response': res
         });

so indeed `eventData.productInfo` we are using is not available in that version.
We will fix that soon.

**Comment by a-laurowski** [comment]
@simonrl  the PR is out. We've changed the way we extract the product id from event data to be compatible with Magento 2.3
Can you take a look on that?

**Comment by simonrl** [comment]
Hi @a-laurowski 

I tested it and it seems to work. Thanks for the fast solution!

-- Simon

**Comment by a-laurowski** [comment]
Fixed in [v2.3.0](https://github.com/FACT-Finder-Web-Components/magento2-module/releases/tag/v2.3.0)
